pr_watcher_notifier/notification.py

```diff
@@ -54,4 +52,15 @@ def send_notifications(data):
         'modified_files': data['modified_files'],
         'pr': pr_data,
     }
-    send_email(context)
+    return context
+
+
+def send_notifications(data):
+    """
+    Send email notifications.
+    """
+    mail = make_email(data)
+    current_app.logger.info('Sending email to {!r} with subject {!r}'.format(
+        context['to'], subject,
```

Comment:
@nedbat, `context`, `subject`, `msg` are all undefined in this function and `mail` overrides the imported name `mail`. I merged this PR as-is and fixed this issue in https://github.com/open-craft/pr_watcher_notifier/commit/373e50096bff8c1703f6c24fb4ab79d1c67a6114#diff-698cdbf4edeae54b454478102d068341 to avoid another review cycle :slightly_smiling_face:

I have set up CI checks for this repository to automatically run the unit tests and also catch code quality issues. :slightly_smiling_face: